pkg/api/routes.go

```diff
@@ -123,7 +123,37 @@ func (h *Handler) GetRoutes() []Route {
 			"ListBuckets",
 			"GET",
 			"/objectstore/buckets",
-			h.Listbuckets,
+			h.ListBuckets,
+		},
+		{
+			"CreateBucket",
+			"POST",
+			"/objectstore/buckets",
```

Comment:
It looks like there is debate one which way is better and both are in use.  I don't think either is right or wrong but personally I think the plural form makes more sense.  http://www.restapitutorial.com/lessons/restfulresourcenaming.html